extensions/spring-scheduled/deployment/src/main/java/io/quarkus/spring/scheduling/deployment/SpringSchedulingProcessor.java

```diff
@@ -0,0 +1,175 @@
+                        if (annotationsValuesContain(springAnnotationValues, "initialDelay")) {
+                            throw new IllegalStateException(
+                                    "Invalid @Scheduled method '" + method.name()
+                                            + "': 'initialDelay' not supported for cron triggers");
+                        }
+                        confValues.add(buildCronParam(springAnnotationValues));
+                    }
+
+                }
+                AnnotationInstance regularAnnotationInstance = AnnotationInstance.create(QUARKUS_SCHEDULED,
+                        scheduledAnnotation.target(), confValues);
+                schedules.add(regularAnnotationInstance);
+            }
+            if (schedules != null) {
+                scheduledBusinessMethods.produce(new ScheduledBusinessMethodItem(bean, method, schedules));
+                LOGGER.debugf("Found scheduled business method %s declared on %s", method, bean);
+            }
+        }
+    }
+
+    AnnotationValue buildCronParam(List<AnnotationValue> springAnnotationValues) {
+        String cronExpression = getAnnotationValueByName(springAnnotationValues, "cron").get().asString();
+        cronExpression = cronExpression.replace("${", "{");
+        return AnnotationValue.createStringValue("cron", cronExpression);
+    }
+
+    AnnotationValue buildEveryParam(List<AnnotationValue> springAnnotationValues) {
+        long fixedRate = getLongValueFromParam(springAnnotationValues, "fixedRate");
+        String fixedRateString;
+        fixedRateString = Duration.ofMillis(fixedRate).toString();
+        AnnotationValue every = AnnotationValue.createStringValue("every", fixedRateString);
+        return every;
+    }
+
+    List<AnnotationValue> buildDelayParams(List<AnnotationValue> springAnnotationValues) {
+        List<AnnotationValue> confValues = new ArrayList<>();
+        long delay = getLongValueFromParam(springAnnotationValues, "initialDelay");
+        confValues.add(AnnotationValue.createLongValue("delay", delay));
+        confValues.add(AnnotationValue.createEnumValue("delayUnit",
+                DotName.createSimple("java.util.concurrent.TimeUnit"),
+                TimeUnit.MILLISECONDS.name()));
+        return confValues;
+    }
+
+    private long getLongValueFromParam(List<AnnotationValue> springAnnotationValues, String paramName) {
+        long paramValue = 0;
+        String paramValueString = "";
+        if (annotationsValuesContain(springAnnotationValues, paramName)) {
+            paramValue = getAnnotationValueByName(springAnnotationValues, paramName).get().asLong();
+        } else { //param value as String e.g. a placeholder ${value.from.conf}
+            paramValueString = getAnnotationValueByName(springAnnotationValues, paramName + "String")
+                    .get().asString();
+            paramValueString = paramValueString.replace("${", "{").trim();
```

Comment:
What does this do?

Reply:
It reads the value from `fixedRateString` and `initialDelayString` params in Spring `@Scheduled`. This params are used to read values from `application.properties`

Reply:
I see, thanks! Do we have tests to verify both kinds of behavior?

Also, we need to note that in Quarkus this support is for build-time properties, not runtime properties

Reply:
Yes, we have tests for both of them.
Ok, I note it for the documentation.